content/en/docs/contributing/docs.md

```diff
@@ -16,10 +16,151 @@ You can add entire sections if you think a topic is missing or update existing o
+The problems can be avoided by building the webside inside a Docker environment.
+If you choose to build the site natively, make sure you use [`Hugo v0.98.0`](https://github.com/gohugoio/hugo/releases/tag/v0.98.0).**
+
+After building and deploying the site, `hugo` provides instructions on accessing and using it:
+
+```text
+Web Server is available at http://localhost:1313/ (bind address 127.0.0.1)
+```
+
+Access the `http://localhost:1313/` URL in the browser to view the site.
+
+#### Using the linter
+
+To automate a big part of the reviewing process, we use the [super linter](https://github.com/github/super-linter) on the `docs` repository.
+It runs as a GitHub action on every time you do a push, checking all the changed files.
+
+{{< img
+   class="w-auto mx-auto"
+   src="/assets/imgs/github-super-linter.png"
+   title="Checking the super linter output"
+   caption="If you open a pull request, you will be able to see the linter output under the checks section."
+   position="center"
+>}}
+
+If you click on `details`, you can see the full linter output.
+The most relevant part is under the `Lint Code Base` section, where you can find the full error messages and the line that generated the error.
+
+{{< img
+   class="w-auto mx-auto"
+   src="/assets/imgs/github-super-linter-output.png"
+   title="Checking the super linter output"
+   caption="Full linter output. Error was caused by line 79 in file `content/en/blog/2022-12-03-random-device-support-in-unikraft.md`"
+   position="center"
+>}}
+
+Since the linter checks the entire file, you may find yourself with a lot of errors from lines that you did not modify.
+If you have the time to add a commit to your pull request that fixes all the linting errors within the modified files, we will be forever grateful :pray:.
+If not, please make sure that the errors are not caused by the lines that you added or modified.
+
+**You can also run the linter locally.**
+For this, you will need to install the [`markdown-cli` tool](https://github.com/igorshubovych/markdownlint-cli).
+
+```console
+$ npm install -g markdownlint-cli
+```
+
+To run the linter, from inside the directory you want to check, use:
+
+```console
+$ markdownlint --config <path-to-your-docs-repo-clone>/.github/workflows/config/config.json --rules <path-to-your-docs-repo-clone>/.github/workflows/rules/rules.js *.md
+```
+
+### Reviewing changes
+
+If you are assigned as a reviewer to a pull request, please follow the [testing steps](docs/contributing/docs/#testing-changes) and make sure everything looks good.
+
+The review may go through a back-and-forth between the authors and the reviewer before the reviewer marks the PR as approved.
+As a reviewer, when you consider the process complete you must add a signature, in the same way that the author has.
+This is done by writing adding a `Reviewed-by` Git trailer like so:
+
+```text
+Reviewed-by: Your Name <your@email.com>
+```
```

Comment:
I think there should be somebody automatically assigned when a PR is made. @StefanJum @razvand do you know if we can do this the same way we have on other Unikraft's repos?

Reply:
Yes, we could do that via the [`governance` scripts](https://github.com/unikraft/governance/), but I wouldn't prioritize this.

Reply:
I agree this is nice to have, but not a priority, since there are not that many prs on the docs repo being opened at the moment.